cvat-core/src/project-implementation.ts

```diff
@@ -96,9 +98,14 @@ export default function implementProject(projectClass) {
         useDefaultSettings: boolean,
         targetStorage: Storage,
         customName?: string,
+        options?: { requestStatusCallback?: (request: Request) => void },
     ) {
-        const result = exportDataset(this, format, saveImages, useDefaultSettings, targetStorage, customName);
-        return result;
+        const { requestStatusCallback } = options || {};
+        const rqID = await exportDataset(this, format, saveImages, useDefaultSettings, targetStorage, customName);
+        if (rqID) {
+            return requestsManager.listen(rqID, { callback: requestStatusCallback });
+        }
```

Comment:
Basically `exportDataset`, from this implementaiton will always return rq id or throw an exception.
So, we do not need this `if` condition. Just update typings for `exportDataset`